website/docs/docs/dbt-versions/versionless-cloud.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Upgrade to \"Keep on latest version\" in dbt Cloud"
+sidebar_label: "Upgrade to \"Keep on latest version\" "
+description: "Learn how to go versionless in dbt Cloud. You never have to perform an upgrade again. Plus, you'll be able to access new features and enhancements as soon as they become available. "
+---
+
+dbt Cloud is going versionless. Soon, your environments and jobs will always run on the latest version of dbt.
+
+This will require you to make one final update to your current jobs and environments. When that's done, you never have to think about managing, coordinating, or upgrading dbt versions again.
```

Comment:
```suggestion
This will require you to make one final update to your current jobs and environments. When that's done, you'll never have to think about managing, coordinating, or upgrading dbt versions again.
```